### ISSUE TITLE
[Feature] Ignore Frontmatter buggy

### ISSUE THREAD

**Issue by Stanzilla** [DavidAnson/vscode-markdownlint#102]
Currently 

```markdown
---
id: patch_notes
name: Patch Notes
title: Patch Notes
---

# Alpha Update 1: January 21th 2024
```

causes a MD025 violation.

`MD025/single-title/single-h1: Multiple top level headings in the same document`

**Comment by DavidAnson** [comment]
That is expected if you have not configured things otherwise. The “title“ property and the H1 heading both serve as titles for the document. Can you please share your configuration?

Relevant documentation:
- https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
- https://github.com/DavidAnson/markdownlint/blob/master/README.md#optionsfrontmatter

**Comment by Stanzilla** [comment]
Hrm okay. My config:

```json
{
    "default": true,
    "MD004": false,
    "MD013": { "line_length": 500, "code_blocks": false, "tables": true },
    "MD029": { "style": "ordered" },
    "MD007": { "indent": 4 },
    "MD046": false,
    "no-alt-text": true,
    "no-inline-html": false,
    "no-space-in-code": false,
    "code-block-style": { "style": "consistent" },
    "no-blanks-blockquote": false
}
```

**Comment by DavidAnson** [comment]
Yes, something like what you suggest! I’m not sure the empty string will work, but honestly anything that does not match should work for what you want.

**Comment by Stanzilla** [comment]
Awesome, thanks! I ran into https://github.com/DavidAnson/vscode-markdownlint/issues/103 which is why I thought it does not work but it does, after a reload.
